packages/babel-core/src/transformation/plugin.js

```diff
@@ -77,6 +77,10 @@ export default class Plugin extends Store {
 
     this.maybeInherit(loc);
 
+    if (this.raw._c) {
```

Comment:
@loganfsmyth Not since `this.raw` will be a `Plugin` instance from another version of Babel. I remember for some reason checking `constructor.name` wouldn't work, and we can't rely on `name` due to minification too.